code/game/machinery/computer/camera_console.dm

```diff
@@ -160,55 +134,18 @@
 		if(!selected_camera)
 			return TRUE
 
-		update_active_camera_screen()
+		SEND_SIGNAL(src, COMSIG_CAMERA_SET_TARGET, selected_camera, selected_camera.view_range, selected_camera.view_range)
 
 		return TRUE
 
-/obj/structure/machinery/computer/cameras/proc/update_active_camera_screen()
-	// Show static if can't use the camera
-	if(!current?.can_use())
-		show_camera_static()
-		return
-
-	// Is this camera located in or attached to a living thing, Vehicle or helmet? If so, assume the camera's loc is the living (or non) thing.
-	var/cam_location = current
-	if(isliving(current.loc) || isVehicle(current.loc))
-		cam_location = current.loc
-	else if(istype(current.loc, /obj/item/clothing/head/helmet/marine))
-		var/obj/item/clothing/head/helmet/marine/helmet = current.loc
-		cam_location = helmet.loc
-
-	// If we're not forcing an update for some reason and the cameras are in the same location,
-	// we don't need to update anything.
-	// Most security cameras will end here as they're not moving.
-	var/newturf = get_turf(cam_location)
-	if(last_camera_turf == newturf)
-		return
-
-	// Cameras that get here are moving, and are likely attached to some moving atom such as cyborgs.
-	last_camera_turf = get_turf(cam_location)
-
-	var/list/visible_things = current.isXRay() ? range(current.view_range, cam_location) : view(current.view_range, cam_location)
-
-	var/list/visible_turfs = list()
-	for(var/turf/visible_turf in visible_things)
-		visible_turfs += visible_turf
-
-	var/list/bbox = get_bbox_of_atoms(visible_turfs)
-	var/size_x = bbox[3] - bbox[1] + 1
-	var/size_y = bbox[4] - bbox[2] + 1
-
-	cam_screen.vis_contents = visible_turfs
-	cam_background.icon_state = "clear"
-	cam_background.fill_rect(1, 1, size_x, size_y)
 
 /obj/structure/machinery/computer/cameras/ui_close(mob/user)
 	var/user_ref = WEAKREF(user)
 	var/is_living = isliving(user)
 	// Living creature or not, we remove you anyway.
 	concurrent_users -= user_ref
 	// Unregister map objects
-	user.client.clear_map(map_name)
+	SEND_SIGNAL(src, COMSIG_CAMERA_UNREGISTER_UI, user)
 	// Turn off the console
 	if(length(concurrent_users) == 0 && is_living)
 		current = null
```

Comment:
Our current is nulled out, but the camera manager's current is not nulled out.
```suggestion
		current = null
		SEND_SIGNAL(src, COMSIG_CAMERA_CLEAR)
```
Without this, a closed camera_console when re-opened is in a weird state where it thinks we aren't currently viewing a camera but the view is on a camera: 
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/69770739-2113-4833-b10c-82184a31356a)

Though I will note that fixing this behavior sort of makes the other bug where it is a incorrectly sized map moot.